docs/_includes/Statements.md

```diff
@@ -658,12 +715,13 @@ PrintStmt =
 ````
 
 The `print` statement is used to print the values of a comma-separated
-list of expressions to the console. The generated C\# code uses
-the `System.Object.ToString()` method to convert the values to printable
+list of expressions to the console. The generated code uses
+target-language-specific idioms to perform this printing, such as 
+the `System.Object.ToString()` method in C\# to convert the values to printable
 strings. The expressions may of course include strings that are used
```

Comment:
Delete the "such as ..." part of the sentence

Reply:
Done. Also added wording about no overriding.